packages/block-editor/src/components/inserter/style.native.scss

```diff
@@ -1,13 +1,32 @@
 /** @format */
 
-.addBlockButton {
+.inserter-menu__add-block-button {
+	border-radius: 16px;
+	margin: 6px 12px;
+	background-color: $blue-wordpress;
+}
+
+.inserter-menu__add-block-button-icon {
 	color: $blue-wordpress;
 }
 
-.addBlockButtonDark {
+.inserter-menu__add-block-button-icon-dark {
```

Comment:
minor nitpick: `dark` is a modifier so if we are sticking with BEM naming the selector should be `inserter-menu__add-block-button--dark`

Reply:
Thanks Jason! I've gone ahead to update this in https://github.com/WordPress/gutenberg/pull/39726/commits/b0693a7c688387407a394b4d026f82a8f03a1686.